siddhar_iyal/star_names.fab

```diff
@@ -0,0 +1,474 @@
+# Kumbam (Kbm)
```

Comment:
The numbers at the end of each line refer to the file references.fab. The numbers shown here come from the Modern skyculture and do not fit to your references.fab file. If possible, add numbers referring to your references. If not, delete these numbers.